dotcom-rendering/src/lib/discussion.ts

```diff
@@ -300,7 +303,7 @@ const discussionApiSuccessSchema = object({
 		isClosedForRecommendation: boolean(),
 		isThreaded: boolean(),
 		title: string(),
-		comments: array(union([commentSchema, replySchema])),
+		comments: array(union([replySchema, commentSchema])),
```

Comment:
Not sure why I flipped these… it should not be necessary, but in any case there’s a test now, so it should not matter too much!